CHARTER.md

```diff
@@ -10,33 +10,37 @@ The Arcalot Round Table *(ART)* consists of:
 
 ## Voting
 
-Each member of the Arcalot Round Table is eligible for voting if they have been a member of the ART for at least 30 days. For any decision that impacts the whole project or processes, a vote is started where a 66% majority with a quorum of 66% of all ART members is needed for the proposal to pass.
+Each member of the Arcalot Round Table is eligible for voting if they have been a member of the ART for at least 30 days. For any decision that impacts the whole project or processes, a vote is started where a minimum of 4 votes is required with 75% approval majority among the voters.
 
-If the majority cannot be reached, the proposal is considered rejected and can be brought to the ART again 2 months after the rejection at the earliest. If the same decision has been rejected twice, it cannot be brought to the ART again.
+If the majority cannot be reached, the proposal is considered rejected and can be brought to the ART again 1 month after the rejection at the earliest. If the same decision has been rejected twice, it cannot be brought to the ART again.
 
-Any voting period must last at least 7 days to account for different time zones and other obligations, and should last no longer than 30 days. The voting period should be stated in the proposal.
+Any voting period must last at least 7 calendar days to account for different time zones and other obligations and should last no longer than 30 days. The voting period should be stated in the proposal. Proposals can be extended no more than twice and for no longer than 30 days total. Extensions can be made by the original submitter, their direct representative, or one of the ART chairs by updating the PR with a note stating the number of calendar days for the extension.
 
-ART members cast their votes by starting a review that either approves or rejects the decision suggested via a pull request. Once the voting period is over or all ART members have voted, the chairs record the decision, objections, and other relevant information in the `art-decisions` folder. Silence equals abstaining from a vote.
+ART members cast their votes by starting a review that either approves or rejects the decision suggested via a pull request. A reject vote is recorded by using the *Request changes* function of the PR review. Once the voting period is over or all ART members have voted, the chairs record the decision, objections, and other relevant information in the [`art-decisions` folder](art-decisions). Silence equals abstaining from a vote.
```

Comment:
Again, very indirect. The proposal is a PR. Members vote by either approving the PR or by requesting changes to the PR. (This has the extra implication, and might be made explicit, that one should provide some detail in *why* they're voting against it and, ideally, what would make them change their mind if that's possible. This was an extremely useful rule in POSIX ... a "no" vote was ignored as trivially malicious if you couldn't describe why and what if anything would make you change your mind.)

Reply:
The update still doesn't provide any guidance/rules on the content of a "request for change". Is anyone willing to consider placing "responsiveness" requirements? E.g., you can't just say "no" ... you're required to suggest substantive content changes that would allow you to change your vote, or to provide a substantive description of why the proposal is fundamentally unacceptable.

Reply:
I don't think it's valid to propose a substantive content change to a proposal PR once it is in the voting phase. By the charter, collaboration and revision should happen while the PR is in draft state. So is it not clear enough to simply require that the "no" vote (as a Request changes review) has an answer to the question of why you are voting no?

Reply:
> So is it not clear enough to simply require that the "no" vote (as a Request changes review) has an answer to the question of why you are voting no?

The distinction is that I like the idea of requiring not only the negative of saying why you don't like it, but the more positive tack of saying what would make you change your mind. It doesn't have to be a detailed proposal or anything, just something that provides a direction towards completion.